bitnami/nginx/values.yaml

```
@@ -274,6 +274,9 @@ serviceAccount:
   ## Only used if `create` is `true`.
   ##
   annotations: {}
+  ## @param serviceAccount.autoMount Auto-mount the service account token in the pod
+  ## 
```

There's a trailing space here that the linter will not like 😅
```suggestion
  ##
```

---

Fixed both, thanks for looking into it.